TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
@schellap if that is how you'd like to handle servicing for the host packages it is fine by me.  Would you like to submit a separate PR to make host "servicing ready" in release branch then I can just commit after that?